ext/gmp/gmp.c

```diff
@@ -28,12 +28,28 @@
 #include "zend_exceptions.h"
 
 #include <gmp.h>
-#include "gmp_arginfo.h"
 
 /* Needed for gmp_random() */
 #include "ext/standard/php_rand.h"
 #include "ext/standard/php_lcg.h"
 
+#define GMP_ROUND_ZERO      0
+#define GMP_ROUND_PLUSINF   1
+#define GMP_ROUND_MINUSINF  2
+
+#ifdef mpir_version
+#define GMP_MPIR_VERSION_STRING  (char *) mpir_version
+#endif
+#define GMP_VERSION_STRING  (char *) gmp_version
```

Comment:
```suggestion
#define GMP_MPIR_VERSION_STRING  ((char *) mpir_version)
#endif
#define GMP_VERSION_STRING  ((char *) gmp_version)
```
Not strictly necessary, but may be better style.